blocks/api/serializer.js

```diff
@@ -208,11 +212,13 @@ export function getCommentDelimitedContent( rawBlockName, attributes, content )
 		serializeAttributes( attributes ) + ' ' :
 		'';
 
-	// strip core blocks of their namespace prefix
+	// Strip core blocks of their namespace prefix.
 	const blockName = startsWith( rawBlockName, 'core/' ) ?
 		rawBlockName.slice( 5 ) :
 		rawBlockName;
 
+	// @todo make the `wp:` prefix potentially configurable.
```

Comment:
Should it be configurable? Or should we just change it per #4636 ? 😄

Reply:
I don't think we should change it :)